applications/nrf5340_audio/src/bluetooth/ble_hci_vsc.c

```diff
@@ -12,12 +12,32 @@
 #include <zephyr/logging/log.h>
 LOG_MODULE_DECLARE(ble, CONFIG_BLE_LOG_LEVEL);
 
-enum ble_hci_vs_max_tx_power {
-	BLE_HCI_VSC_MAX_TX_PWR_0dBm = 0,
-	BLE_HCI_VSC_MAX_TX_PWR_3dBm = 3,
-};
+int ble_hci_vsc_set_fem_pin(struct ble_hci_vs_cp_set_fem_pin *fem_pin)
```

Comment:
usually the "set" part of the name comes last.

Reply:
Changed for all the functions in ble_hci_vsc